### DISCUSSION TITLE
求助：安装后无法启动前端

### DISCUSSION THREAD

**Discussion by HaorongX** [hydro-dev/Hydro#362] [Q&A] [answered]
我正在fedora系统上部署hydro，手动安装了hydro和依赖项。在连接了mongodb后用yarn debug --port 2333启动hydro，随后网页显示如下图
![网页捕获_22-5-2022_11210_localhost](https://user-images.githubusercontent.com/103825064/169676409-d34615e0-48fa-491b-a5bc-52a604735a6c.jpeg)

而且我无法使用hydrooj命令，我是否漏掉了什么步骤？我该怎么做？

**Comment by wuxianucw** [comment, marked as answer]
由于你使用 `yarn debug` 启动，我推测你正在使用开发模式部署，在此情况下，`hydrooj` 命令须用 `npx hydrooj` 替代。出现图示状况的原因是 `ui-default` 插件没有加载，该插件在开发模式下默认不加载。

**Reply by HaorongX** [reply]
感谢。我刚刚添加了ui-default插件，前端已经可以正常显示。
